addons/web_editor/static/src/scss/web_editor.common.scss

```diff
@@ -237,6 +237,11 @@ img.ms-auto, img.mx-auto {
     width: auto;
 }
 
+// We don't want to have bold on font-size classes by default.
+.h1, .h2, .h3, .h4, .h5, .h6, .display-1, .display-2, .display-3, .display-4 {
+    font-weight: inherit;
```

Comment:
Indeed :+1: Use web_editor.frontend.scss I guess @Guillaume-gdi ... web_editor.common.scss contains a lot of wrong stuff I think but let's not add more :wink:

Reply:
I cannot put my scss rules in the fontend scss because I need it also in the backend :|.. For example, when a user edits a task description in the project app, `web_editor.frontend.scss` is not loaded.

With the change of the spec, I will only customize new classes in this file so I suppose it's OK.